cmd/mysterium_server/command_run/command.go

```diff
@@ -62,6 +64,10 @@ func (cmd *CommandRun) Run(options CommandOptions) (err error) {
 		SourceAddress: "10.8.0.0/24",
 		TargetIp:      vpnServerIp,
 	})
+
+	// clear probable stale entries
+	cmd.NatService.Stop()
```

Comment:
That's already implemented in command stop  function `cmd.Kill()`

Reply:
Yes, but it is not called. And we need just to clear stale iptables entries.

Reply:
Still have question how to handle that